tests/test_dse.py

```diff
@@ -487,10 +487,12 @@ def test_full_shape(self, rotate):
                                               'cire-rotate': rotate}))
 
         # Check code generation
+        bns, pbs = assert_blocking(op1, {'x0_blk0'})
+
```

Comment:
ok in this special case, since this whole block of code is checking codegen, maybe here you don't need a blank line after all